src/features/DevTools/slice.ts

```diff
@@ -0,0 +1,40 @@
+import { createAction, createReducer } from '@reduxjs/toolkit';
+import { call, takeEvery } from 'redux-saga/effects';
+
+import { path } from '@vendor';
+
+const sliceName = 'demo';
+const initialState = {
+  greeting: 'Hello from Reducer',
+  count: 0
+};
+
+/* Selectors */
+export const getGreeting = path([sliceName, 'greeting']);
+export const getCount = path([sliceName, 'count']);
```

Comment:
I'm not sure what the benefit is of using these selectors over writing a simple function (like `state => state.demo.greeting`). These selectors also return a value with the type `unknown`.

Reply:
I'm still experimenting with them and suspect that the syntax can become cumbersome once all the reducers are combined. The type-safety is a problem though. I'll look into a better solution once I have a real store.